### ISSUE TITLE
New Pipe to conditionally-pluralize words

### ISSUE THREAD

**Issue by cpilson** [danrevah/ngx-pipes#130]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[X] feature request
```

* **Ngx-pipes version:** `2.3.6`

**Current behavior**
Want to make something plural here via a pipe? No luck. :(

**Expected behavior**
Wouldn't it be great to pass in a context, like 'Picture', and have a pipe that could automatically pluralize this for you?

Example in-template use: `<span>Viewing {{filteredRecords}} of {{recordCount}} {{Picture | make-plural-string: totalRecords}}</span>` would output something like `Viewing 0 of 0 Pictures` or `Viewing 1 of 1 Picture` or `Viewing 10 of 20 Pictures`.

**Minimal reproduction of the problem with instructions**
Not a bug, but if you've ever worked with pagination or record counting, and have become fed up of one-offing all of these, this is a good pipe to have.

**Please tell us about your environment:**
macOS, npm, Angular, Node.

* **Angular version:** `6.0.9`
```
Angular CLI: 6.0.8
Node: 8.10.0
OS: darwin x64
Angular: 6.0.9
```

* **Node (for AoT issues):** `node --version` =   `v8.10.0`

**Comment by cpilson** [comment]
```
remote: Permission to danrevah/ngx-pipes.git denied to cpilson.
fatal: unable to access 'https://github.com/danrevah/ngx-pipes.git/': The requested URL returned error: 403
```

:(

**Comment by danrevah** [comment]
Released in @ v2.3.9
